contrib/systemd/spacenavd.service

```diff
@@ -1,12 +1,10 @@
 [Unit]
 Description=3Dconnexion Input Devices Userspace Driver
-After=syslog.target
 
 [Service]
-Type=forking
+Type=exec
 PIDFile=/var/run/spnavd.pid
-ExecStart=/usr/local/bin/spacenavd
-StandardError=syslog
+ExecStart=/usr/local/bin/spacenavd -d -l /dev/stdout
```

Comment:
-d -- don't daemonize; systemd does not require you to daemonize
-l /dev/stdout -- put the logs on stdout. They will be captured from there and stderr and put where configured.